src/System.Runtime.Intrinsics/ref/System.Runtime.Intrinsics.cs

```diff
@@ -1139,802 +1131,802 @@ public abstract class X64
         public static uint MultiplyNoFlags(uint left, uint right) { throw null; }
-        public static unsafe uint MultiplyNoFlags(uint left, uint right, uint* low) { throw null; }
+        public unsafe static uint MultiplyNoFlags(uint left, uint right, uint* low) { throw null; }
         public static uint ParallelBitDeposit(uint value, uint mask) { throw null; }
         public static uint ParallelBitExtract(uint value, uint mask) { throw null; }
-        public abstract class X64
+        public static uint ZeroHighBits(uint value, uint index) { throw null; }
+        public abstract partial class X64
         {
             internal X64() { }
             public static bool IsSupported { get { throw null; } }
-            public static ulong ZeroHighBits(ulong value, ulong index) { throw null; }
             public static ulong MultiplyNoFlags(ulong left, ulong right) { throw null; }
-            public static unsafe ulong MultiplyNoFlags(ulong left, ulong right, ulong* low) { throw null; }
+            public unsafe static ulong MultiplyNoFlags(ulong left, ulong right, ulong* low) { throw null; }
             public static ulong ParallelBitDeposit(ulong value, ulong mask) { throw null; }
-            public static ulong ParallelBitExtract(ulong value, ulong mask) { throw null; } 
+            public static ulong ParallelBitExtract(ulong value, ulong mask) { throw null; }
+            public static ulong ZeroHighBits(ulong value, ulong index) { throw null; }
         }
     }
     public enum FloatComparisonMode : byte
     {
-        EqualOrderedNonSignaling = 0,
-        LessThanOrderedSignaling = 1,
-        LessThanOrEqualOrderedSignaling = 2,
-        UnorderedNonSignaling = 3,
-        NotEqualUnorderedNonSignaling = 4,
-        NotLessThanUnorderedSignaling = 5,
-        NotLessThanOrEqualUnorderedSignaling = 6,
-        OrderedNonSignaling = 7,
-        EqualUnorderedNonSignaling = 8,
-        NotGreaterThanOrEqualUnorderedSignaling = 9,
-        NotGreaterThanUnorderedSignaling = 10,
-        FalseOrderedNonSignaling = 11,
-        NotEqualOrderedNonSignaling = 12,
-        GreaterThanOrEqualOrderedSignaling = 13,
-        GreaterThanOrderedSignaling = 14,
-        TrueUnorderedNonSignaling = 15,
-        EqualOrderedSignaling = 16,
-        LessThanOrderedNonSignaling = 17,
-        LessThanOrEqualOrderedNonSignaling = 18,
-        UnorderedSignaling = 19,
-        NotEqualUnorderedSignaling = 20,
-        NotLessThanUnorderedNonSignaling = 21,
-        NotLessThanOrEqualUnorderedNonSignaling = 22,
-        OrderedSignaling = 23,
-        EqualUnorderedSignaling = 24,
-        NotGreaterThanOrEqualUnorderedNonSignaling = 25,
-        NotGreaterThanUnorderedNonSignaling = 26,
-        FalseOrderedSignaling = 27,
-        NotEqualOrderedSignaling = 28,
-        GreaterThanOrEqualOrderedNonSignaling = 29,
-        GreaterThanOrderedNonSignaling = 30,
-        TrueUnorderedSignaling = 31,
+        EqualOrderedNonSignaling = (byte)0,
```

Comment:
As a note: The generator ordered the enum alphabetically. It would likely be significantly better to order it numerically.

Reply:
Logged https://github.com/dotnet/arcade/issues/1487